core.py

```diff
@@ -277,7 +277,7 @@ def text_style(s, style, always=False):
         # No color for pipes, files and Pythonista console
         if not always and (
         	isinstance(sys.stdout, StringIO)
-        	or isinstance(sys.stdout, file)
+        	or isinstance(sys.stdout, IOBase)
```

Comment:
The second parameter of [__isinstance()__](https://docs.python.org/2/library/functions.html#isinstance) can be a tuple as in __isinstance(sys.stdout, (StringIO, IOBase))__.

However you do not even need that trick here because StringIO is inherited from IOBase...
```
$ python3
>>> import io
>>> sio = io.StringIO()
>>> isinstance(sio, io.IOBase)
True
```